go.mod

```diff
@@ -69,6 +69,7 @@ require (
 	github.com/spf13/cast v1.4.1 // indirect
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/subosito/gotenv v1.2.0 // indirect
+	github.com/ugorji/go/codec v1.2.7 // indirect
 	github.com/valyala/fasthttp v1.34.0 // indirect
 	github.com/xeipuuv/gojsonpointer v0.0.0-20180127040702-4e3ac2762d5f // indirect
```

Comment:
would you please update oapi version from github.com/deepmap/oapi-codegen v1.9.1 version to v1.10.0? 

A release recently was https://github.com/deepmap/oapi-codegen/releases/tag/v1.10.1 :) 

v1.10.0 contains protobuf version update https://github.com/deepmap/oapi-codegen/pull/553

Reply:
sure :) probably will not couple that with this diff.

Reply:
done